### ISSUE TITLE
Proposal:  Enhanced Course Card Design in Open edX Dashboard

### ISSUE THREAD

**Issue by asma-ahmedd** [openedx/platform-roadmap#355]
### Abstract

This initiative proposes UI/UX improvements for the Open edX dashboard's course card design. By applying principles of visual perception and information processing, the goal is to enhance discoverability of courses, improve comprehension of key information, and increase learner engagement through a more user-friendly interface.

### Context & Background

The current course card design presents information in a way that can be visually cluttered and difficult to scan. Learners struggle to quickly find relevant courses or identify important details like course end dates. This initiative aims to address these pain points and improve the overall user experience.

### Scope & Approach

The primary persona for this initiative is the learner who interacts with the Open edX dashboard. The approach focuses on optimizing the information layout within the course card. This includes:

- Separating organization name, course name, and end date for improved readability.
- Integrating icons to enhance visual engagement and information processing speed.
- Utilizing typography and visual hierarchy to prioritize essential information.
- Maintaining consistent placement and styling across all course cards for a cohesive experience.

**A detailed analysis of the proposal with supporting research can be found here: [Wiki Proposal Link](https://openedx.atlassian.net/wiki/spaces/OEPM/pages/4193484824/Proposal+Enhanced+Course+Card+Design+in+Open+edX+Dashboard)**

### Value & Impact

The success of this initiative will be measured by:

- Increased efficiency in finding relevant courses.
- Improved user comprehension of key information like course end dates.
- Higher learner engagement through a more visually appealing interface.

### Milestones and/or Epics

**Milestone 1:** Course Card Information Redesign

- Implement separation of organization name, course name, and end date.
- Integrate icons for course name and potentially other elements.
- Refine typography and visual hierarchy for optimal readability.
- Measure user interaction time and success rate in finding desired courses.

### Named Release

Unsure

### Timeline

To be determined

### Proposed By

Arbisoft

### Additional Info

https://openedx.atlassian.net/l/cp/70SkWoTL

**Comment by jmakowski1123** [comment]
Tag Liz C at Schema

**Comment by asma-ahmedd** [comment]
@ali-hugo @lizc577 when you have a moment, could you both please take a look at this project proposal?

**Comment by ali-hugo** [comment]
@asma-ahmedd Sure! Is there a date by which you'd like to have feedback? I'm working on a pretty tight deadline at the moment...

**Comment by asma-ahmedd** [comment]
@ali-hugo Thanks, Ali! Whenever it's convenient for you is fine, but could you please let me know a date by which you think you might be able to provide your feedback? I really appreciate it!

**Comment by ali-hugo** [comment]
@asma-ahmedd I found some time today to take a look at the proposal. It looks great! I left some feedback [here](https://openedx.atlassian.net/wiki/spaces/OEPM/pages/4193484824?focusedCommentId=4255023112). I wasn't able to upload an image to the wiki comments, so I'll add it to this ticket instead:

<img width="974" alt="card buttons" src="https://github.com/openedx/platform-roadmap/assets/88187719/2e57cec0-caee-420c-959d-4f582613f30b">

**Comment by jmakowski1123** [comment]
cc @marcotuts for Liz to take a quick look

**Comment by MaferMazu** [comment]
@asma-ahmedd I don't remember if this is the card we should move to on hold. Is it? And if you have more context, it will be helpful.

**Comment by asma-ahmedd** [comment]
@MaferMazu Yes, this card should be on hold. We're currently waiting to hear back from the Bizdev team to confirm if the project is moving forward. If it does, @cassiezamparini and I will be working on it together.
